packages/cli/configuration/src/generators-yml/generatorInvocations.ts

```diff
@@ -35,7 +35,7 @@ export const GENERATOR_INVOCATIONS: Record<GeneratorName, Omit<GeneratorInvocati
         version: "0.7.2"
     },
     [GeneratorName.TYPESCRIPT_NODE_SDK]: {
-        version: "0.9.5",
+        version: "0.13.0",
```

Comment:
@bsinghvi lets do `0.12.x` since we know there are some bugs merged into `0.13.x`